modules/devices/storage.c

```diff
@@ -24,11 +24,13 @@
 
 gchar *storage_icons = NULL;
 
+// udisks2 support by Ondrej Čerman
```

Comment:
You can add the credits to the file header next to the copyrights.

Reply:
Moved